snuba/web/rpc/common/common.py

```diff
@@ -195,11 +195,14 @@ def get_subscriptable_field(field: Expression) -> SubscriptableReference | None:
 
 
 def attribute_key_to_expression(attr_key: AttributeKey) -> Expression:
+    def _build_label_mapping_key(attr_key: AttributeKey) -> str:
+        return attr_key.name + " " + AttributeKey.Type.Name(attr_key.type)
```

Comment:
tip: don't use spaces as delimiters in software systems. `__` usually is a better choice

Reply:
https://github.com/getsentry/snuba/pull/6592